dlt-connector/src/graphql/schema.ts

```diff
@@ -0,0 +1,13 @@
+import { Decimal } from 'decimal.js-light'
+import { GraphQLSchema } from 'graphql'
+import { buildSchema } from 'type-graphql'
+
+import { DecimalScalar } from './scalar/Decimal'
+import { TransactionResolver } from './resolver/TransactionsResolver'
+
+export const schema = async (): Promise<GraphQLSchema> => {
+  return buildSchema({
+    resolvers: [TransactionResolver],
```

Comment:
I use the current Apollo Version 4, not the deprecated Apollo Version 2 which is used from the rest project.
https://www.apollographql.com/docs/apollo-server/v2
Apollo Server 2 is officially deprecated, with end-of-life scheduled for 22 October 2023. Additionally, certain features are end-of-life as of 31 December 2023

In the current Apollo Version a string array is not longer allowed for BuildSchemaOptions.resolvers only NonEmptyArray<Function>
so your code don't longer work. I have copied schema.ts from backend and had to adjust it, because it don't longer work.